src/components/CampaignDetails.vue

```diff
@@ -13,15 +13,15 @@
     </section>
       <div class="contribution-area">
         <h3>Make a Contribution</h3>
-        <p>Minimum Contribution  <b> {{ campaign.min_contribution }}</b></p>
-        <input v-model='contribution' placeholder='Enter your contribution' type="number"/>
+        <p>To become an approver of this campaign please contribute more than:
+          <b> {{ ethMinContribution() }} ETH</b></p>
+        <input v-model='contribution' placeholder='Enter your contribution (ETH)' type='number' step='0.0001' min='0'/>
         <button :disabled="!contribution"
         @click="submitContribution">
         Submit Contribution </button>
-        <p class="user-message"
-          v-if="userMessage">
-          Please enter a contribution over {{campaign.min_contribution}}!
-        </p>
+        <button :disabled="!contribution"
+        @click="convertToDollars">
+        Convert To USD </button>
```

Comment:
This button doesn't do anything yet but thought I would add it to make sure we don't forget.